From 09b368ec313908fcae0de2a7e3c401f539268af4 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Wed, 3 Sep 2025 15:46:46 +0200
Subject: [PATCH] arm64: dts: st: usb2phy: add clock names on stm32mp211

dt-bindings define clock-names, add the core clock name available on
stm32mp211 usb2phy.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I7b3fccd5d6d994634927d53f8727b4fea91c68a9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/487223
Reviewed-by: Christian BRUEL <christian.bruel@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp211.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp211.dtsi b/arch/arm64/boot/dts/st/stm32mp211.dtsi
index ef95ba2264b7..bedb23303b68 100644
--- a/arch/arm64/boot/dts/st/stm32mp211.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp211.dtsi
@@ -258,6 +258,7 @@ usb2_phy1: usb2-phy1 {
 		#clock-cells    = <1>;
 		st,syscfg       = <&syscfg 0x2400>;
 		clocks          = <&rcc CK_KER_USB2PHY1>;
+		clock-names     = "core";
 		resets          = <&rcc USB2PHY1_R>;
 		status          = "disabled";
 	};
@@ -268,6 +269,7 @@ usb2_phy2: usb2-phy2 {
 		#clock-cells    = <1>;
 		st,syscfg       = <&syscfg 0x2800>;
 		clocks          = <&rcc CK_KER_USB2PHY2EN>;
+		clock-names     = "core";
 		resets          = <&rcc USB2PHY2_R>;
 		interrupts-extended = <&exti1 44 IRQ_TYPE_EDGE_RISING>;
 		wakeup-source;
-- 
2.53.0

